dev/InfoBar/TestUI/Themes/InfoBarStyles.xaml

```diff
@@ -3,6 +3,12 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:local="using:MUXControlsTestApp">
 
+    <SolidColorBrush x:Key="InfoBarSeverityCriticalStatusColor" Color="#D13438"/>
+    <SolidColorBrush x:Key="InfoBarSeverityWarningStatusColor" Color="#FF8C00"/>
```

Comment:
Can these colours be moved to the top level ResourceDictionary so the same colours can be used for the Form Control validation coming in the future?

Reply:
Yes, good idea. I've made a note of that and will address them in the next PR.